### ISSUE TITLE
Theory for website

### ISSUE THREAD

**Issue by comane** [HEP-PBSP/colibri#338]
## Important

- [x] Theory predictions: how they are built and snippets of code to exemplify (eg np.einsum(...))
- [x] Likelihood function description: including base + all penalty terms
- [x] Bayesian priors: list all options and also discuss bayesian update here.
- [x] MC loss functions
- [x] Inference methods: discuss the theory of analytical fits (linear regression), MC replica fits (~bootstrap method + gradient descent) and Bayesian fits
